Automattic/wp-calypso
# Start: Skip submit event if in plans

Fixes https://github.com/Automattic/wp-calypso/issues/97646

## Proposed Changes

We now skip firing `calypso_signup_actions_submit_step (step:domains)` when we click on "Continue with Free Plan" in the Plans step in Start.

<img width="498" alt="image" src="https://github.com/user-attachments/assets/11b94a76-4aaa-4cd5-b156-c15db729aa48" />


## Why are these changes being made?


*

## Testing Instructions



* Live link and go to /start
* Custom domain
* Free plan
* Click on "Continue with Free Plan"
* No `calypso_signup_actions_submit_step (step:domains)` should be fired in the Plans step
